doc/yarp_env_variables.md

```diff
@@ -93,6 +92,7 @@ Deprecated Environmental Variables
 | Environmental variable        | Description | Related documentation page |
 |:-----------------------------:|:-----------:|:--------------------------:|
 | `YARP_POLICY`                 |  Legacy variable used to change the behavior of the ResourceFinder. Deprecated since YARP 2.3.65  | |
+| `YARP_STACK_SIZE`             | Default stack size (in bytes) for YARP threads. Deprecated since YARP 3.0.0 | |
```

Comment:
Deprecated or removed? If it is removed, I think we can remove it from the table.

Reply:
:+1:  removing it later in a commit of cleanup of the documentation

Reply:
Removed, because it is no longer an available option with c++11 threads